indico/htdocs/js/indico/jquery/colorswitch.js

```diff
@@ -0,0 +1,114 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+(function($) {
+    $.widget("indico.colorswitch", {
+
+        options: {
+            defaultColors: [
+                [
+                    {bgColor: '#EEE0EF', textColor: '#1D041F'},
+                    {bgColor: '#E3F2D3', textColor: '#253F08'},
+                    {bgColor: '#FEFFBF', textColor: '#1F1F02'},
+                    {bgColor: '#DFE555', textColor: '#202020'},
+                    {bgColor: '#FFEC1F', textColor: '#1F1D04'}
+                ], [
+                    {bgColor: '#DFEBFF', textColor: '#0F264F'},
+                    {bgColor: '#0D316F', textColor: '#EFF5FF'},
+                    {bgColor: '#1A3F14', textColor: '#F1FFEF'},
+                    {bgColor: '#5F171A', textColor: '#FFFFFF'},
+                    {bgColor: '#D9DFC3', textColor: '#272F09'}
+                ], [
+                    {bgColor: '#4F144E', textColor: '#FFEFFF'},
+                    {bgColor: '#6F390D', textColor: '#FFEDDF'},
+                    {bgColor: '#8ec473', textColor: '#021F03'},
+                    {bgColor: '#92b6db', textColor: '#03070F'},
+                    {bgColor: '#DFDFDF', textColor: '#151515'}
+                ], [
+                    {bgColor: '#ecc495', textColor: '#1F1100'},
+                    {bgColor: '#b9cbca', textColor: '#0F0202'},
+                    {bgColor: '#C2ECEF', textColor: '#0D1E1F'},
+                    {bgColor: '#d0c296', textColor: '#000000'},
+                    {bgColor: '#EFEBC2', textColor: '#202020'}
+                ]
+            ]
```

Comment:
`, onClickCallback: null` (otherwise it's not really clear that this exists).

And I'd call it `onSelect` (datepicker also uses this name iirc.)